styles/style.css

```diff
@@ -1,387 +1,305 @@
-/* universal styles */
 html {
   box-sizing: border-box;
   font-family: "Montserrat", sans-serif;
   font-size: 16px;
   color: rgb(255, 255, 255);
   background-color: black;
 }
+
 *,
 *:before,
 *:after {
   box-sizing: inherit;
   padding: 0;
   margin: 0;
 }
+
 img {
   display: block;
   max-width: 100%;
 }
 
-/* mobile */
-/* landing-section */
-.container-1-pc,
-.box-container,
-.container-2-pc {
-  display: none;
-}
-
-.landing-section {
-  height: 100vh;
-  width: 100%;
-  background-image: url(/images/balkouras-nicos--tJcdoXAtlQ-unsplash.jpg);
-  background-position: center;
-  background-size: cover;
-  display: flex;
-}
-
-.mobile {
-  margin: auto;
-  text-align: center;
-}
-
-.mobile h1 {
-  padding: 1rem;
+a {
+  text-decoration: none;
+  color: white;
 }
 
-#mobile {
+a:hover {
+  cursor: pointer;
   text-decoration: underline;
-  font-weight: 900;
-  font-size: 2.3rem;
+  color: red;
 }
-/* tablet */
-@media screen and (min-width: 768px) {
-  .landing-section {
-    flex-direction: column;
-    justify-content: space-between;
-  }
-  .mobile {
-    display: none;
-  }
-  .container-1-pc,
-  .box-container,
-  .container-2-pc {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 2rem;
-  }
 
-  .landing-section h1 {
-    font-size: 2.5rem;
-  }
+:root {
+  --dvh-with-fallback: 100vh;
+  --padding: 2rem;
+  --padding-big: 5rem;
 
-  .container-1-pc div,
-  .container-2-pc div {
-    width: 33%;
-  }
+  --font-size-heading: 3rem;
+  --font-weight-heading: 800;
 
-  #about-us h1,
-  #sign-up h1 {
-    float: right;
-  }
+  --font-size-subheading: 2rem;
 
-  .box-horizontal {
-    width: 20rem;
-    height: 0.5rem;
-    background-color: aliceblue;
-  }
+  --font-size-landing: 3.7rem;
+  --font-weight-landing: 700;
 
-  .box-vertical {
-    width: 0.5rem;
-    height: 20rem;
-    background-color: aliceblue;
-  }
+  --font-size-paragraph: 1.5rem;
+  --font-weight-paragraph: 600;
 
-  a {
-    text-decoration: none;
-    color: white;
-  }
+  --font-size-landing-mobile: 2.3rem;
+  --font-weight-landing-mobile: 800;
+
+  --font-size-paragraph-mobile: 1.2rem;
+  --font-weight-paragraph-mobile: 500;
 
-  a:hover {
-    cursor: pointer;
-    text-decoration: underline;
-    color: red;
-  }
 }
 
-/* pc */
-@media screen and (min-width: 1024px) {
-  .landing-section h1 {
-    font-size: 3.7rem;
-  }
+@supports (height: 100dvh) {
 
-  a:hover {
-    cursor: pointer;
+  /*only executed if the browser suppots it*/
+  :root {
+    --dvh-with-fallback: 100dvh;
```

Comment:
Check out dvh and svh, they are the better version of vh, not fully supported though..